lib/resources/commonTypes.tsx

```diff
@@ -154,20 +158,24 @@ export type SearchExtras = {
 };
 
 // Info for distribution bar.
-export type Distribution = {
+export type UserDistribution = {
```

Comment:
if Distribution is a template straight out of the major object, UserDistribution traces back to user and their progress